examples/multi-router/index.js

```diff
@@ -6,7 +6,7 @@ app.use('/api/v1', require('./controllers/api_v1'));
 app.use('/api/v2', require('./controllers/api_v2'));
 
 app.get('/', function(req, res) {
-  res.send('Hello form root route.');
+  res.send('Hello from root route.');
```

Comment:
Can you strip the semicolon from this line?

We are slowly transitioning to follow standard style, and whenever we touch a line we update that line to follow it 👌